packages/controller/src/resources/monitoring/system/dashboards/opstrace-overview.json

```diff
@@ -348,7 +205,7 @@
       "timeFrom": null,
       "timeRegions": [],
       "timeShift": null,
-      "title": "[LOGS] API proxy /loki/api/v1/push, POSTs, rate by status code (5m mean)",
+      "title": "[LOGS] API proxy /loki/api/v1/push, POSTs, rate by status code (5m mean) ALL tenants",
```

Comment:
:rocket:

Reply:
FWIW, I have now added a panel just like this that is "`BY tenants`", i.e., it is broken down by tenant rather than summed across them.  Please let me know how useable this is, especially the title of the panel.